compose/backend.yml

```diff
@@ -202,6 +202,8 @@ services:
       MICROSERVICEKEYS_API_GW: "${IDAM_KEY_API_GW}"
       MICROSERVICEKEYS_HMC_HMI_INBOUND_ADAPTER: "${IDAM_KEY_API_HMI_INBOUND_ADAPTER}"
       MICROSERVICEKEYS_CCD_CASE_DISPOSER: "${IDAM_KEY_CCD_CASE_DISPOSER}"
+      MICROSERVICEKEYS_IDAM_KEY_TS_TRANSLATION_SERVICE: "${IDAM_KEY_TS_TRANSLATION_SERVICE}"
+      MICROSERVICEKEYS_TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET: "${TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET}"
```

Comment:
This is the wrong place for the TS IDAM client value if it is required.

Reply:
done